grouper/fe/handlers.py

```diff
@@ -717,6 +718,136 @@ def get(self, group_id=None, name=None):
         )
 
 
+class AuditsComplete(GrouperHandler):
```

Comment:
This is what they're called, however; the organization performs an internal audit, and then it exposes the results to the external auditors.

I mean, I don't care too much, but it at least aligns the two processes so it's clear they're related?